web/viewer.js

```diff
@@ -902,6 +902,9 @@ var PDFViewerApplication = {
           var left = store.get('scrollLeft', '0');
           var top = store.get('scrollTop', '0');
 
+          self.pageRotation = store.get('pageRotation', '0');
+          self.pdfViewer.pagesRotation = self.pageRotation;
```

Comment:
Unfortunately this won't work, since it breaks the thumbnails (and it's probably also related to the console error).
I believe that you need to use `PDFViewerApplication.rotatePages` instead, see [viewer.js#L1326](https://github.com/mozilla/pdf.js/blob/master/web/viewer.js#L1326).